### PR TITLE
Show model

### PR DESCRIPTION
**Pull request by Kornelia-K**

### CODE REVIEW

**Comment by BohdanOne** [comment]
Please make pull requests to branch dev only!

File: models/show.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+
+const showSchema = new mongoose.Schema({
+        movieTitle:{
+            type: String,
```
Comment (BohdanOne) [inline models/show.js]: here we will need reference to actual movie in our database (so not a string but an id)

File: models/show.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        },
+        duration:{
+            type: Number,
+        },
+        room:{
```
Comment (BohdanOne) [inline models/show.js]: are we going to have more than one room?

File: models/show.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+            type: Number,
+        },
+        date: {
+            type: String,
```
Comment (BohdanOne) [inline models/show.js]: there is a typa Date which is more suitable in this situation

File: models/show.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        }
+
```
Comment (BohdanOne) [inline models/show.js]: we also need to store here informations about reservations made for that particular show (so, array of id's) and information about available seats

File: models/show.js (hunk 2 above)
Comment (Kornelia-K) [reply models/show.js]: I don't know, but I think that in the most cinemas are more that one room.

File: models/show.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ const mongoose = require('mongoose');
 
 const showSchema = new mongoose.Schema({
         movieTitle:{
-            type: String,
+            type: [mongoose.Schema.Types.ObjectId], ref: 'Movie',
```
Comment (BohdanOne) [inline models/show.js]: this should be reference to one movie so why an array?
and we don't need unique property (actually it is possible we will have few shows of the same movie)

File: models/show.js [hunk 6]
```diff
@@ -18,6 +18,14 @@ const showSchema = new mongoose.Schema({
             type: Date,
             required: true,
         }
+        available_seats: {
```
Comment (BohdanOne) [inline models/show.js]: we need to know which seats exactly are available not only the overall number of them

File: models/show.js (hunk 2 above)
Comment (BohdanOne) [reply models/show.js]: ok, but all the code we write needs to be consistent in whole app. was there any discussion about number of rooms with other team members? adding more than one room will bring another level of complexity to many parts of our app

**Comment by BohdanOne** [comment]
after making corrections, please close this pr and make another to branch dev

File: models/show.js (hunk 2 above)
Comment (Kornelia-K) [reply models/show.js]: no, there wasn't any discussion yet. I will talk about it after christmas break.

**Pull request closed without merging**